### PR TITLE
improved exmaple

### PR DESCRIPTION
**Pull request by mikrowelt**
fixed bugs
improved exmaple

1. bind oauth2.openedtest.com to localhost
2. run fcr: branch - oauth-screens_85627684, port 9000
3. change api server stanging in fcr config
4. run a basic http server and serve opened-oauth.js repo on port 4200 and branch improvments-fix1
5. change oauth.js file localy and change it apiHost to staging and openedHost to http://local.opened.io:9000
6. change index.html
set ur client_id to 66d0354299f860b4e39c56979f5cfd7d5216f67ce958c19ad5a3a68e6b0a2399
and redirect_uri to http://oauth2.openedtest.com:4200/callback.html
7. open in browser oauth2.openedtest.com:4200
8. follow ui

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,22 +6,85 @@
...
 		window.OpenEd.api.init({
-		  client_id: '7e922c66bf453f1ab30646cc154c4e58d96ae770e828ce3d208b5f1a94ceb53e',
-		  redirect_uri: 'http://oauth2.openedtest.com:4200'
+		  client_id: '66d0354299f860b4e39c56979f5cfd7d5216f67ce958c19ad5a3a68e6b0a2399',
+		  redirect_uri: 'http://oauth2.openedtest.com:4200/callback.html'
 	    });
 	    window.OpenEd.api.login(function () {
-		  window.OpenEd.api.request('/current_user.json', null, function () {
-		    console.log(arguments);
+		  window.OpenEd.api.request('/current_user.json', null, function (data) {
```
Comment (LeonidMorozov) [inline index.html]: Please, use `/users/me.json` endpoint instead.

File: oauth.js [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
     isInited: false,
 
-    apiHost: 'https://openedengine-sandbox1.herokuapp.com',
+    apiHost: 'https://api-staging.opened.io',
```
Comment (LeonidMorozov) [inline oauth.js]: Not sure it's a good idea to point example app to our api-staging server. @larsburgess , what do you think? Is it OK?

Comment (larsburgess) [inline oauth.js]: No, it should point to prod. In fact, why even have them configure it? It
should be baked into the library and you can override it as an advanced use
case.

File: index.html [hunk 3]
```diff
@@ -6,22 +6,118 @@
...
-		  client_id: '7e922c66bf453f1ab30646cc154c4e58d96ae770e828ce3d208b5f1a94ceb53e',
-		  redirect_uri: 'http://oauth2.openedtest.com:4200'
+		  client_id: '66d0354299f860b4e39c56979f5cfd7d5216f67ce958c19ad5a3a68e6b0a2399',
+		  redirect_uri: 'http://oauth2.openedtest.com:4200/callback.html'
 	    });
 	    window.OpenEd.api.login(function () {
-		  window.OpenEd.api.request('/current_user.json', null, function () {
-		    console.log(arguments);
+		  window.OpenEd.api.request('/users/me.json', null, function (data) {
+            console.log(data);
```
Comment (artursmirnov) [inline index.html]: Please remove `console.log`

**Pull request merged**